collections/_infographics/energy/electricity-per-capita-world.md

```diff
@@ -14,4 +14,29 @@ data-orig:
   - [ "UN (population)", "https://population.un.org/wpp/Download/Files/1_Indicators%20(Standard)/EXCEL_FILES/1_Population/WPP2019_POP_F01_1_TOTAL_POPULATION_BOTH_SEXES.xlsx" ]
 ---
 
-{% include includes-local/comment-placeholder.md slug="infografiky/elektrina-na-osobu-svet" %}
+## How to read this graph
+
+* The graph shows the **worldwide development of electricity produced per person by individual sources in the years 2000-2019**, as well as a comparison with selected world regions. Values are given in **<glossary id="w">kWh</glossary> of electricity produced per person per year**.
+* All graphs have the same vertical axis, so for regions with low average consumption per person, the filled part occupies only a minimal area of the graph.
+* However, the per capita comparison does not account for the different sizes of each region, so **the right column completes the regions' share of total global electricity generation in 2019**. The supplementary infographic, [Electricity generation in world regions](/infographics/electricity-per-capita-world), compares regions in terms of total electricity generation.
+
+## Comments on the graph
+
+* **Electricity production per person in the Czech Republic in 2019 was 8,066 kWh, significantly higher than the averages in the regions shown, except for the USA.** The Czech Republic is also a significant exporter of electricity, so the average electricity consumption per person was lower, at 6,891 kWh. Even this value is more than double the world average.
+* Globally, electricity production increased by 75% between 2000 and 2019, while at the same time, the population has grown by 1.6 billion people. n per capita terms, electricity generation increased by 39%, and in absolute terms, there was an increase in the capacity of all electricity generation sources except nuclear. The share of coal in electricity production fell slightly (from 37.9% to 35.2%), and the percentage of nuclear power fell even more significantly (from 16.9% to 10.5%). In contrast, there was an increase in the share of natural gas (from 18.2 % to 23.5%) and renewable sources (from 19.5% to 27.3% due to the development of wind and solar power plants).
+* **Trends of individual regions** observed from the graph:
+  * In the **European Union**, electricity consumption is virtually unchanged per capita, but there is a significant substitution of coal for renewables in electricity generation.
+  * Electricity consumption per person in the **USA** has fallen slightly but is still many times higher than the world average. Since 2007, there has been a decrease in the share of coal in electricity production, which is, however, being replaced mainly by natural gas, partly also by renewable sources.
```

Comment:
change "gas, partly also" to "gas, as well as partly"

Reply:
Updated.